### ISSUE TITLE
[Bug Report][3.1.12] Setting default color on VRadio has no effect

### ISSUE THREAD

**Issue by abea** [vuetifyjs/vuetify#17052]
### Environment
**Vuetify Version:** 3.1.12
**Vue Version:** 3.2.47
**Browsers:** Firefox 111.0
**OS:** Mac OS 10.15

### Steps to reproduce
Set the `color` prop for VRadio in the `createVuetify` defaults object.
```
    VRadio: {
      color: 'primary'
    }
```

### Expected Behavior
There is no color change.

### Actual Behavior
The color should apply to the radio icons unless a prop is set in the component file.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqdUstO3DAU/RXLm4A0iQWtVGk0VHTT7lvEBrMIyc3EJbGta2cAjfLvXD8YheGxQNnk+J5z7vNmz3cTeNU9Vf8dX3M1WoOe7VmDUHu4TjE2sw7NyIrMLaSWGh4jtTHaeZYD7OK18GQvNWO+hxHWZNpCV0+Dv0q4GNS29wWbV4GUY454ATJ2/bdulQmyxgwGiQ+IBotEZ0yITPmDZrJLntKdybRZ6vlUar7iv6ytqErqceNhtAMV+TMwNruytjb+RjDWShOSPnyEMaQotyFHJmViDLChvoPhQvLfCp2XnO3qYQLCXcIiWn2g+Qc0vHYhcvnhU9VVr7BlJ33tmEVjTxd6H0KE4yAOfjU+JctsKN5rKrzm1hNIQ9mIxbAIugaV9cyBj8rDuSB0yyOJB7IRiU08mn+iUg5Ll2Y07SHuWeaAk/yweRluMmDJe++tWwsxaXu/rRozikuKCZy0VyOUrRkvv1Xn1fcfolXOL98rcGN5h+bBAVJGyfPZRPNwmx8lyGHyPavOzpNvfitpBy4YvzEUlGUHWCLoFhDw0+qPuK86OIq96eLlqGmig9L37miYjYuDvHkp7QvtRYugvw1p5hVP5VHCJOG3z2O4aCg=)

### Other comments
We can set a radio button color by setting the `color` prop on VRadio. However, setting that prop with a default in the `createVuetify` `defaults` object has no effect. It does work to set the default on VRadioGroup, however.

**Comment by jonasgadeikis** [comment]
Hello, any updates on this issue? I am migrating a large scale project to Vuetify 3 and I encountered this issue aswell.

**Comment by jonasgadeikis** [comment]
I have investigated this issue further to see if I can fix this and I found that in VRadio component code the actual rendered component is VSelectionControl. And if you add a default color for VSelectionControl like this:

```js
VSelectionControl: {
    color: 'primary',
},
```

This does work and your VRadio gets the primary color without passing a color prop when using it in template. I think this issue can be closed.

**Comment by abea** [comment]
I wouldn't close the issue based on that. The documentation doesn't speak to sub-components in this way and the `color` property is part of the VRadio API. As of now the expectation set by the documentation is that any prop could be configured in this way. If that's not correct then the documentation should be updated to set more clear expectations.

**Comment by joaobarcia** [comment]
I agree with @abea 's comment. The defaults object is very useful but has been quite unintuitive to work with.

**Comment by m3nu** [comment]
This works too for setting the color:

```
VRadioGroup: { color: 'primary' }
```

Unfortunately I couldn't make it work for `true-icon`, even after trying a few variations and icon name formats.

```
VSelectionControl: { 'true-icon': 'fas dot-circle' }  
```

My aim would be to replace this:

```
<v-radio true-icon="fas fa-dot-circle" ...>
```

**Comment by J-Sek** [comment]
Fixed since v4.0.0 (probably by [`2a74859`](https://github.com/vuetifyjs/vuetify/commit/2a74859))
